code/__DEFINES/conflict.dm

```diff
@@ -93,6 +93,11 @@
 #define GUN_SHOWS_AMMO_REMAINING (1<<14) //Whether the mob sprite reflects the ammo level
 #define GUN_SHOWS_LOADED (1<<15) //Whether the mob sprite as loaded or unloaded, a binary version of the above
 #define GUN_SMOKE_PARTICLES (1<<16) //Whether the gun has smoke particles
+///Can only fire forwards
+#define GUN_FORWARD_FIRE_ANGLE (1<<17)
+
+///degree of cone in front of which a gun with forward fire angle flag can fire
+#define GUN_FIRE_CONE_ALLOWED 120
```

Comment:
why dont we just make this a var on the gun? would eliminate both these defines

Reply:
changed to vars instead of defines.